tools/pip_install.bat

```diff
@@ -4,28 +4,28 @@ ECHO "Starting setup..."
 :ontology_validator_setup
 ECHO "Starting ontology validator setup"
 CD validators\ontology_validator
-START py -m pip install .
+START python3 -m pip install .
```

Comment:
The problem is not the version though. The problem is using the windows `py` python launcher fails to correctly install dependencies on windows machines. I want to directly call the python executable instead of using the python launcher.

However, I don't know why there is a discrepancy...